qiita_pet/handlers/analysis_handlers.py

```diff
@@ -141,14 +142,17 @@ def get(self, analysis_id):
             jobres[jobject.datatype].append((jobject.command[0],
                                              jobject.results))
 
-        dropped = {}
         dropped_samples = analysis.dropped_samples
+        dropped = defaultdict(list)
         if dropped_samples:
-            for proc_data_id, samples in viewitems(dropped_samples):
+            for proc_data_id, samples in viewitems(analysis.dropped_samples):
```

Comment:
In fact, the `if` statement on line 147 is not needed here, since this `for` loop will not run if `dropped_samples` is empty